Mark9Mbugua/Politico-API
# #163680705 Get a specific political office

#### What does this PR do?
It gets a specific political office

#### Description of Task to be completed?
Have the endpoint below working:
GET api/v1/offices/<int:id>

#### How should this be manually tested?
- Clone this repo in a local repo
- Make sure that all the necessary requirements have been installed
- Flask run the application
- On Postman, test GET api/v1/offices/<int:id>

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2243191
#163680705
